pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -416,114 +464,136 @@ func TestReconcileUpdate(t *testing.T) {
 	username := "johnsmith"
 	namespaceName := "toolchain-member"
 
-	t.Run("update dev to advanced tier", func(t *testing.T) {
-		// given
-		nsTmplSet := newNSTmplSet(namespaceName, username, "advanced", withNamespaces("dev"))
-		// create namespace (and assume it is complete since it has the expected revision number)
-		devNS := newNamespace("basic", username, "dev", withRevision("abcde11"))
-		rb := newRoleBinding(devNS.Name, "user-edit")
-		r, req, fakeClient := prepareReconcile(t, namespaceName, username, nsTmplSet, devNS, rb)
+	t.Run("success", func(t *testing.T) {
+		t.Run("update dev to advanced tier", func(t *testing.T) {
+			// given
+			nsTmplSet := newNSTmplSet(namespaceName, username, "basic", withNamespaces("dev"), withClusterResources())
```

Comment:
is there any benefit of setting the NSTemplateSet tier to `basic` and then update it to `advanced` both as part of the test setup? It doesn't have any impact on the test, but looks a bit more confusing (and needs more code).

Reply:
yes, I've noticed it as well and removed it